test/cleanup

```diff
@@ -2,4 +2,8 @@
 
 source $(dirname $0)/common
 
-ursula "envs/test" ${ROOT}/playbooks/${TEST_ENV}/tasks/delete.yml
+if [ -f ${ROOT}/envs/test/ssh.config ]; then
+    URSULA_SSH_CONFIG="--ursula-ssh-config=\"${ROOT}/envs/test/ssh.config\""
```

Comment:
ursula-cli already has code to opportunistically lad an ssh config from the environment directory. 
https://github.com/blueboxgroup/ursula-cli/blob/master/ursula_cli/shell.py#L361-L363

Have you tested with that? Might work if you change the file name from `ssh.config` to `ssh_config`.